homeassistant/components/switchbot_cloud/vacuum.py

```diff
@@ -58,11 +73,23 @@ async def async_setup_entry(
     VACUUM_FAN_SPEED_MAX: "3",
 }
 
+VACUUM_FAN_SPEED_TO_SWITCHBOT_FAN_SPEED_V2: dict[str, str] = {
+    VACUUM_FAN_SPEED_QUIET: "1",
```

Comment:
Differences between different vacuum models should be handled in the library and not directly in HA.

Reply:
done